### PR TITLE
CORE-41 Graceful destruction of SpitfireRtc and RtcConductor instances

### PR DESCRIPTION
**Pull request by roman380**

### CODE REVIEW

File: Spitfire/webrtc.props [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    <WebRtcOut>$(WebRtcSrc)\out\$(Platform)\$(Configuration)</WebRtcOut>
+  </PropertyGroup>
+  <PropertyGroup />
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);$(WebRtcSrc)\third_party\abseil-cpp;$(WebRtcSrc)</AdditionalIncludeDirectories>
+    </ClCompile>
+    <Link>
+      <AdditionalLibraryDirectories>%(AdditionalLibraryDirectories);$(WebRtcOut)\obj</AdditionalLibraryDirectories>
+      <AdditionalDependencies>%(AdditionalDependencies);webrtc.lib</AdditionalDependencies>
```
Comment (Codeusa) [inline Spitfire/webrtc.props]: Why are we changing library loading here when `Spitfire.vcxproj` already contains ` <AdditionalDependencies>webrtc.lib;winmm.lib; Secur32.lib;%(AdditionalDependencies)</AdditionalDependencies>`

File: Spitfire/RtcConductor.cpp [hunk 2]
```diff
@@ -51,7 +51,7 @@ namespace Spitfire
 					x.second->dataChannel = nullptr;
 				}
 			}
-			dataObservers.empty();
+			//dataObservers.empty();
```
[the remark this answers is not in the document]
Comment (roman380) [reply Spitfire/RtcConductor.cpp]: But it's `clear` not `empty`. I re-added this in a later commit though.

File: Spitfire/webrtc.props (hunk 1 above)
Comment (roman380) [reply Spitfire/webrtc.props]: I cleared settings references nd combined them in single [.props file](https://github.com/RainwayApp/spitfire/blob/CORE-41/Spitfire/webrtc.props) where they are better visible including both .h and .lib.

**Comment by roman380** [comment]
BTW where do you see `m80`? I can only see `m79` [there](https://webrtc.googlesource.com/src)

**Comment by Codeusa** [comment]
> BTW where do you see `m80`? I can only see `m79` [there](https://webrtc.googlesource.com/src)

https://chromiumdash.appspot.com/branches shows all the branches. It is currently just unnamed at 3987

**Comment by roman380** [comment]
> > BTW where do you see `m80`? I can only see `m79` [there](https://webrtc.googlesource.com/src)
> 
> https://chromiumdash.appspot.com/branches shows all the branches. It is currently just unnamed at 3987

Nice, I hope to remember this link because once in a while it will be useful again.

**Pull request merged**